Tests/SwiftDocCTests/Model/ParametersAndReturnValidatorTests.swift

```diff
@@ -111,6 +111,62 @@ class ParametersAndReturnValidatorTests: XCTestCase {
         }
     }
     
+    func testExtendsReturnValueDocumentation() throws {
+        for (returnValueDescription, expectsExtendedDocumentation) in [
+            ("Returns some value.", true),
```

Comment:
Can we add some more cases here for when we expected extended documentation? Such as using similar words to the one we're searching for but which should not trigger the extended documentation:
> failsafe
> error-prone